apps/examples/nettest/nettest.c

```diff
@@ -807,6 +872,8 @@ int nettest_main(int argc, char *argv[])
 			tcp_server_thread(num_packets_to_process);
 		} else if (proto == NT_UDP) {
 			udp_server_thread(num_packets_to_process);
+		} else if (proto == NT_BROADCAST) {
```

Comment:
Currently branches here is very simple but you had better use switch-case if there are more than 2 else if 
It is good habit. Also you already check mode above, below else if(mode == NETTEST_CLIENT_MODE) can be changed just else.

Reply:
I will change all same things in this file via another commit